test-data/local/config/everest/config.ert

```diff
@@ -0,0 +1,14 @@
+NUM_REALIZATIONS 2
```

Comment:
Please rename the `everest` part of the path and remove it from the comment.

Reply:
Done